CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+## Contributing
+
+[![Build Status](https://travis-ci.org/probot/stale.svg?branch=master)](https://travis-ci.org/probot/stale)
+[![Codecov](https://img.shields.io/codecov/c/github/probot/probot.svg)](https://codecov.io/gh/probot/probot/)
+
+[code-of-conduct]: CODE_OF_CONDUCT.md
+[good-first-issue-search]: https://github.com/search?utf8=%E2%9C%93&q=topic%3Aprobot+topic%3Aprobot-app+good-first-issues%3A%3E0&type=
+
+Hi there! We're thrilled that you'd like to contribute to this project. Your help is essential for keeping it great.
+
+Please note that this project is released with a [Contributor Code of Conduct][code-of-conduct]. By participating in this project you agree to abide by its terms.
+
+Want to chat with Stale or Probot users and contributors? [Join us in Slack](https://probot-slackin.herokuapp.com/)!
+
+## Issues and PRs
+
+If you have suggestions for how Stale could be improved, or want to report a bug, open an issue! We'd love all and any contributions. If you have questions, too, we'd love to hear them.
+
+We'd also love PRs. If you're thinking of a large PR, we advise opening up an issue first to talk about it, though! Look at the links below if you're not sure how to open a PR.
+
+## Just starting out? Looking for how to help?
+
+Use [this search][good-first-issue-search] to find Probot apps that have issues marked with the `good-first-issue` label.
+
+## Apps
+
+We have one app enabled on this repo:
+
+- [Stale](https://probot.github.io/apps/stale/): We use Stale (the app created in this repo) to ensure that conversations here remain relevant. This is for us, the maintainers, so that we don't feel like we've got hundreds of issues to deal with; if you still have an issue, please let us know! We don't want to close issues that are painful for you. Stale just helps us have a bit more breathing space by making sure issues don't pile up forever.
+
+If you're concerned about our apps or feel that they are insensitive in some way, please let us know.
```

Comment:
This section is likely to be the same on 99% of projects in this org, so I'd be up for centralizing it in a future PR.

Reply:
👍 Works for me.